### ISSUE TITLE
multilanguage site menu with offcanvas

### ISSUE THREAD

**Issue by nadjak77** [gantry/gantry5#3282]
Hi,
I have a multilanguage site.
In my template I add the menu to have the offcanvas menu too.
But I only can choose one of the menues and they don't change if I swith the language.
In the documentation is explaned to do a layout for every language. Is that right? That is not an option for me.
If this is the only way, I will use a modul position with the standard joomla menue.

**Comment by N8Solutions** [comment]
@nadjak77 Yes, you need to have a layout for each language. This way you can assign a language to it. I don't understand why it isn't an option for you? It is very easy to do, you just need to follow the instructions.

Also, something else to understand, a Gantry 5 Outline is seen by Joomla as a "Template Style". So if you were using a different template, you would still need to duplicate the Template style that you want to use for the other languages. For reference, check out [Step 9 in this article from Themexpert](https://www.themexpert.com/blog/beginners-guide-to-multilingual-joomla-website) where it tells you to select your default template style and  duplicate it and then specify the language it is for. The Gantry 5 Menu particle makes this easy because all you have to do is set it to use the "Active Menu" so when you switch languages it will automatically show the menu for that language. This way you don't have to create menu modules for each language you want to show like you would need to for other templates.

I suggest you look over [the Gantry 5 Documentation again on setting up a multilingual site](https://docs.gantry.org/gantry5/tutorials/multilingual-site) because it works flawlessly if done correctly.

Good luck and hope this helps!
